tests/integration_tests/http_api/util/test_case.py

```diff
@@ -298,6 +300,19 @@ def set_log_level():
 
     print("pass test: set log level")
 
+def get_tso():
+    # test state: all
+    url = V2_BASE_URL0+"/tso"
+
+    for i in range(RETRY_TIME):
```

Comment:
What is the motivation for retrying here? What can happen if we don't retry?

Reply:
loop deleted